client/components/CustomBankField.js

```diff
@@ -0,0 +1,74 @@
+import {has, translate as t} from '../Helpers';
+import T from './Translated';
+
+export default class CustomBankField extends React.Component {
+
+    constructor(props) {
+        has(props, 'params');
+
+        super(props);
+    }
+
+    getValue() {
+        let node = this.refs[this.props.params.name].getDOMNode();
+        return {
+            name: this.props.params.name,
+            value: (this.props.params.type === "number") ? parseInt(node.value, 10) : node.value
+        };
+    }
+
+    setValue(value) {
+        this.props.params.currentValue = value;
```

Comment:
`props` should be (and might be, because of React?) considered immutable. Once again, `setValue` is only called with `""`, so it will do nothing most of the time. Let's just delete it while we have another patch (later) for showing the actual correct values, what do you think?